vcd/resource_vcd_org.go

```diff
@@ -74,6 +74,70 @@ func resourceOrg() *schema.Resource {
 				Default:     true,
 				Description: "True if this organization is allowed to share catalogs.",
 			},
+			"vapp_lease": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				MaxItems:    1,
+				Description: "Defines lease parameters for vApps created in this organization",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"maximum_runtime_lease_in_sec": &schema.Schema{
+							Type:     schema.TypeInt,
+							Optional: true,
+							//Default:      604800, // (7 days)
```

Comment:
Just checking - are these commented out values are what vCD sets if nothing is set?
If so - I could say this could be phrased differently as it gives doubt whether that was commented out code block ar a comment which ment some system defaults

Reply:
Changed comments